acceptance/suites/tests/010-puppetserver-cli/subcommand/irb.rb

```diff
@@ -0,0 +1,40 @@
+require 'test/unit/assertions'
+require 'puppetserver/acceptance/gem_utils'
+
+test_name "Puppetserver 'irb' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
+  cli = "pe-puppetserver"
+else
+  cli = "puppetserver"
+end
+# --------------------------------------------------------------------------
+
+%w(GEM_PATH RUBYLIB RUBYOPT RUBY_OPTS).each do |var|
+  step "Check that #{var} is cleared"
+  cmd = "echo 'puts ENV[%{#{var}] || %{OK}' | #{var}=BAD #{cli} irb -- -f"
+  on(master, cmd) do
+    assert(/^OK$/.match(stdout), "#{var} is not being cleared")
+    assert_no_match(/BAD/, stdout)
+  end
+end
+
+%w(FOO_DEBUG).each do |var|
+  step "Check that #{var} is preserved"
+  cmd = "echo 'puts ENV[%{#{var}}] || %{BAD}' | #{var}=OK #{cli} irb -- -f"
+  on(master, cmd) do
+    assert(/^OK$/.match(stdout), "#{var} is not being preserved")
+    assert_no_match(/BAD/, stdout)
+  end
+end
+
+step "Check that puppet is loadable"
+cmd = "echo 'puts Puppet.version' | #{cli} irb -- -rpuppet -f"
+on(master, cmd) do
+  assert_no_match(/LoadError/, stdout)
```

Comment:
I thought about that, but couldn't quickly come up with one that I was confident would always match.  Do you think `/\..*\./` is sufficient?